app/modules/eus/enduser_setup.html

```diff
@@ -160,6 +160,8 @@ <h3>Connect device to your Wi-Fi</h3>
 				</div>
 				<input id=ssid type=text autocorrect=off autocapitalize=none placeholder='Wi-Fi Name' />
 				<input id=wifi_password type=text autocorrect=off autocapitalize=none autocomplete=off placeholder=Password />
+				<input id=TwitterAPI type=text autocorrect=off autocapitalize=none autocomplete=off placeholder='Twitter API Handler' />
+				<input id=TwitterMessage type=text autocorrect=off autocapitalize=none autocomplete=off placeholder='Twitter Message' />
 				<button id=submit type=button>Save</button>
```

Comment:
I agree with Johny here. I'm a bit surprised to see no changes to the documentation. I wouldn't want to change the default page but see documented how to change it in

- the module documentation
- or by  creating a nice example at https://github.com/nodemcu/nodemcu-firmware/tree/master/lua_examples and linking documentation to it